src/Context/ContextDefinition.php

```diff
@@ -81,7 +81,12 @@ public static function createFromArray($values) {
     // Default to Rules context definition class.
     $values['class'] = isset($values['class']) ? $values['class'] : '\Drupal\rules\Context\ContextDefinition';
     if (!isset($values['type'])) {
-      $values['type'] = 'any';
+      if (!isset($values['value'])) {
+        $values['type'] = 'any';
+      }
+      else {
+        $values['type'] = $values['value'];
+      }
```

Comment:
@klausi -- yeah, we are using values['type'] down stream. I'm can find you the references if you want.  Changing them from 'type' to 'value' should probably be a separate issue, however.